src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -831,6 +837,74 @@ class Browser extends DashboardView {
     });
   }
 
+  showExportSelectedRowsDialog(rows) {
+    this.setState({
+      rowsToExport: rows
+    });
+  }
+
+  cancelExportSelectedRows() {
+    this.setState({
+      rowsToExport: null
+    });
+  }
+
+  async confirmExportSelectedRows(rows) {
+    this.setState({ rowsToExport: null });
+    const className = this.props.params.className;
+    const query = new Parse.Query(className);
+
+    if (rows['*']) {
+      // Export all
+      query.limit(10000);
```

Comment:
I think it would be a good idea to make this configurable (config.json) and default it to 1000 rows. If the database isn't indexed properly this could lead to problems.

Reply:
Good idea, we'll make a note about this as potential future improvement.